CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,76 @@
+# Contributor Covenant Code of Conduct
+
+Agro Millers is committed to providing a safe, respectful and inclusive environment for all its contributors, collaborators, and users. We expect all members of our community to abide by the following code of conduct:
```

Comment:
It seems like you misspelled the name of the application. Please change _AgroMillers_ to **_AgroMillets_** everywhere.

Reply:
My bad. I haven't proofread the file! I have updated the file. You can check it now.